docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1257 @@
+    "\n",
+    "# Method for training the model\n",
+    "def train(trainloader, testloader, n_epochs, patience):\n",
+    "    model = Net()\n",
+    "    criterion = nn.CrossEntropyLoss()\n",
+    "    optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)\n",
+    "    model = model.to(device)\n",
+    "\n",
+    "    best_test_accuracy = 0.0\n",
+    "    for epoch in range(n_epochs):  # loop over the dataset multiple times\n",
+    "        running_loss = 0.0\n",
+    "\n",
+    "        for _, data in enumerate(trainloader, 0):\n",
+    "            # get the inputs; data is a list of [inputs, labels]\n",
+    "            inputs, labels = data[\"image\"].to(device), data[\"label\"].to(device)\n",
+    "\n",
+    "            # zero the parameter gradients\n",
+    "            optimizer.zero_grad()\n",
+    "\n",
+    "            # forward + backward + optimize\n",
+    "            outputs = model(inputs)\n",
+    "            loss = criterion(outputs, labels)\n",
+    "            loss.backward()\n",
+    "            optimizer.step()\n",
+    "\n",
+    "            # print statistics\n",
+    "            running_loss += loss.item()\n",
+    "\n",
+    "        accuracy = get_test_accuracy(model, testloader)\n",
+    "        print(\n",
+    "            f\"epoch: {epoch + 1} loss: {running_loss / len(trainloader):.3f} test acc: {accuracy}\"\n",
+    "        )\n",
+    "\n",
+    "        running_loss = 0.0\n",
+    "\n",
+    "        if accuracy > best_test_accuracy:\n",
+    "            best_epoch = epoch\n",
+    "\n",
+    "        if epoch - best_epoch > patience:\n",
+    "            print(f\"Early stopping at epoch {epoch + 1}\")\n",
+    "            break\n",
+    "    return model\n",
+    "\n",
+    "\n",
+    "# Method for computing out-of-sample embeddings\n",
+    "def compute_embeddings(model, testloader):\n",
+    "    embeddings_list = []\n",
+    "\n",
+    "    with torch.no_grad():\n",
+    "        for data in tqdm(testloader):\n",
+    "            images, labels = data[\"image\"].to(device), data[\"label\"].to(device)\n",
+    "\n",
+    "            embeddings = model.embeddings(images)\n",
+    "            embeddings_list.append(embeddings.cpu())\n",
+    "\n",
+    "    return torch.vstack(embeddings_list)\n",
+    "\n",
+    "\n",
+    "# Method for computing out-of-sample predicted probabilities\n",
+    "def compute_pred_probs(model, testloader):\n",
+    "    pred_probs_list = []\n",
+    "\n",
+    "    with torch.no_grad():\n",
+    "        for data in tqdm(testloader):\n",
+    "            images, labels = data[\"image\"].to(device), data[\"label\"].to(device)\n",
+    "\n",
+    "            outputs = model(images)\n",
+    "            pred_probs_list.append(outputs.cpu())\n",
+    "\n",
+    "    return torch.vstack(pred_probs_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Create K folds of the dataset for cross validation"
```

Comment:
```suggestion
    "## 4. Prepare the dataset for K-fold cross-validation \n",
    "\n",
    "To find label issues based on `pred_probs`, we recommend out-of-sample predictions, which can be produced via K-fold cross-validation. To ensure this tutorial runs quickly, we set K and other important neural network training hyperparameters to small values here. Use larger values to get good results in practice!"
```

Reply:
TODO: hyperlink to this page when we mention "out-of-sample predictions":
https://docs.cleanlab.ai/stable/tutorials/pred_probs_cross_val.html

Look at how the other tutorials do it, to ensure your linking is proper